docs/connectivity/Agent.mdx

```diff
@@ -0,0 +1,275 @@
+---
+id: Agent
+title: Instrument Agents
+displayed_sidebar: webUiSidebar
+---
+
+import { PlusCircleOutlined, 
+         SelectOutlined,
+         DiffOutlined
+       } from '@ant-design/icons';
+
+The Ganymede Agent is an executable that can be installed on lab instruments running Windows or Linux to capture instrument data systematically.  The agent can
```

Comment:
Some lab instruments do run Windows/Linux, but we should say here that the Agent is running on the attached PC.